### ISSUE TITLE
AsciiDoc.ImageContainsAltText is no longer catching images without alt text

### ISSUE THREAD

**Issue by sandbergja** [redhat-documentation/vale-at-red-hat#829]
**Describe the bug**
After #737 was merged, AsciiDoc.ImageContainsAltText does not catch images without alt text in asciidoc.

Examples:

* `image::media/marcoverlay5.png[]` no longer creates an AsciiDoc.ImageContainsAltText alert

**To Reproduce**

1. Create a directory with the following basic .vale.ini:
    ```
    StylesPath = vale-styles

    MinAlertLevel = suggestion

    Packages = AsciiDoc

    [*.adoc]
    AsciiDoc.ImageContainsAltText = error
    ```
1. Add this document to your directory as no_alt.adoc:
    ```
    = No alt text =
    
    image::media/marcoverlay1.png[]
    ```
1. Run `vale sync`
2. Run `vale no_alt.adoc`

**Expected behavior**
I would expect the following:

```
 no_alt.adoc
 3:1  error  Image is missing accessibility  AsciiDoc.ImageContainsAltText
             alt tags.

✖ 1 error, 0 warnings and 0 suggestions in 1 file.
```

**Actual behavior**

I get:

```
✔ 0 errors, 0 warnings and 0 suggestions in 1 file.
```

**Additional context**
From my understanding of the [vale docs](https://vale.sh/docs/topics/styles/#existence), if there are multiple elements in a `raw` array, they are simply concatenated together, meaning we are currently looking for the pattern `image::.*\[\]image:.*\[\]`, which won't find much.

I will open a pull request for this issue shortly.

**Comment by mmcclaingithub** [comment]
I think the addition caught some empty alt text for the second pattern with the single colon when I looked at some docs a couple of weeks ago, but if the change in the PR is more succinct and covers both patterns, that works for me.
